groundhog-th/groundhog-th.cabal

```diff
@@ -22,7 +22,7 @@ library
                    , time                     >= 1.1.4
                    , containers               >= 0.2
                    , unordered-containers     >= 0.1.3
-                   , aeson                    >= 0.7       && < 2
+                   , aeson                    >= 0.7       && < 2.1
```

Comment:
It's fine to put a more permissive upper bound, like `< 3`

Reply:
Not according to PVP - the first two components denote the major version.